packages/styles/forms.css

```diff
@@ -118,6 +137,10 @@ textarea.Field--has-error,
   color: var(--field-border-color-error);
 }
 
+.Icon--checkbox-unchecked.Checkbox__overlay:active.Field--has-error:not(.Checkbox__overlay--disabled) {
```

Comment:
```suggestion
.Checkbox__overlay:active.Field--has-error:not(.Checkbox__overlay--disabled) {
```

This feels like an overly specific selector, can we not just use `.Checkbox__overlay`?

Reply:
Ah in this case, @bobbyomari mentioned to me that it should only show the red active fill state when checking the unchecked box, not when unchecking the checked box

Reply:
If that is the case, we should be checking the checked state and not the class.

Reply:
Something like `.Checkbox__overlay:not(:checked):active...`